.github/dependabot.yml

```diff
@@ -9,3 +9,6 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+              - name: Update Nix Flake Lock
+                uses: DeterminateSystems/update-flake-lock@v21
+            
```

Comment:
This doesn't look schema compliant https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

Reply:
Ya sorry, I have made the changes now, please have a look.

Reply:
You changed the indentation, I think you are confusing GitHub actions yaml files with dependabot.yml. These are two totally different config file types with different schemas (look at the link for available options).